authtools/views.py

```diff
@@ -28,7 +28,7 @@
 from django.shortcuts import redirect, resolve_url
 from django.utils.functional import lazy
 from django.utils.http import base36_to_int, is_safe_url, urlsafe_base64_decode
-from django.utils import six
+import six
```

Comment:
This shouldn't be necessary since we've already upgraded to Python 3

Reply:
It was removed, because Django no longer supports Python 2. `six` is a compatibility library that helps w/ keeping code working in Python 2 and 3 https://pypi.org/project/six/. We only use Python 3 so the use of the package is unnecessary